safe-room.lic

```diff
@@ -193,6 +182,22 @@ class SafeRoom
     bput('lie', 'You lie') if standing?
   end
 
+  def use_pc_empaths?(pc_empaths, settings)
+    return if pc_empaths.empty?
```

Comment:
❓  Should this be `return false`?

Reply:
Yea. im guessing predicates should be explicit about true/false